Bottom: cfa0e77e75c5916658adb6db2edde6e75c0adac2
Top:    6ff4c3fef859b775522c27db8de4a8fcfcf792c3
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2013-02-01 22:30:26 -0800

Refresh of kern-bug

---

diff --git a/src/cma.c b/src/cma.c
index 0220f1f..547781e 100755
--- a/src/cma.c
+++ b/src/cma.c
@@ -384,7 +384,7 @@ static void ucma_insert_id(struct cma_id_private *id_priv)
 		return;
 
 	pthread_mutex_lock(&mut);
-	idm_set(&ucma_idm, id_priv->handle, rs);
+	idm_set(&ucma_idm, id_priv->handle, id_priv);
 	pthread_mutex_unlock(&mut);
 }
